e2e/js_image_oci/repo/BUILD.bazel

```diff
@@ -0,0 +1,29 @@
+load("@aspect_bazel_lib//lib:copy_to_directory.bzl", "copy_to_directory")
+load("@aspect_bazel_lib//lib:copy_to_bin.bzl", "copy_to_bin")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "sources",
+    srcs = [
+        "source.txt",
+        "srcdir/source.txt",
```

Comment:
Any reason to name them the same?

Reply:
no, just wanted to assert that external directories are treated okay.